testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestAvroSchemaEvolution.java

```diff
@@ -26,7 +26,7 @@
 import static io.trino.tests.product.utils.QueryExecutors.onHive;
 import static java.lang.String.format;
 
-public class TestAvroSchemaEvolution
+public class BaseTestAvroSchemaEvolution
```

Comment:
I would expect this commit does not compile correctly as the test class does not start with `Test`. Hence I suggested adding a subclass already in commit which does initial rename.